pymatgen/core/structure.py

```diff
@@ -3642,11 +3641,12 @@ def get_boxed_structure(
         ):
             box_center = [(i + 0.5) * a, (j + 0.5) * b, (k + 0.5) * c]
             if random_rotation:
+                rng = np.random.default_rng()
                 while True:
                     op = SymmOp.from_origin_axis_angle(
                         (0, 0, 0),
-                        axis=np.random.rand(3),
-                        angle=random.uniform(-180, 180),
```

Comment:
Usage of `random` module is removed altogether, as there is no good reason to reply on two different random generators (`numpy` and `random`) at the same time.